src/test/java/racingcar/domainTest/JudgementTest.java

```diff
@@ -12,47 +12,47 @@
 import racingcar.domain.Judgement;
 
 public class JudgementTest extends NsTest {
-    public static final int LENGTH_OF_CARS = 3;
-
-    @Test
-    void 우승자를_가리는_기능() {
-        Judgement judgement = new Judgement();
-        List<Car> carListSoloWin = new ArrayList<>();
-        List<String> cars = new ArrayList<>(Arrays.asList("pobi", "minju", "java"));
-        List<Integer> progressSoloWin = new ArrayList<>(Arrays.asList(1, 2, 1));
-        List<String> winners;
-        String result;
-
-        for (int i = 0; i < LENGTH_OF_CARS; i++) {
-            carListSoloWin.add(new Car(cars.get(i), progressSoloWin.get(i)));
-        }
-
-        winners = judgement.judgeWinner(carListSoloWin);
-        result = String.join(", ", winners);
-        assertThat(result).contains("minju");
-    }
 
-    @Test
-    void 공동우승자를_가리는_기능() {
-        Judgement judgement = new Judgement();
-        List<Car> carListDuoWin = new ArrayList<>();
-        List<String> cars = new ArrayList<>(Arrays.asList("pobi", "minju", "java"));
-        List<Integer> progressDuoWin = new ArrayList<>(Arrays.asList(1, 2, 2));
-        List<String> winners;
-        String result;
-
-        for (int i = 0; i < LENGTH_OF_CARS; i++) {
-            carListDuoWin.add(new Car(cars.get(i), progressDuoWin.get(i)));
-        }
-        winners = judgement.judgeWinner(carListDuoWin);
-        result = String.join(", ", winners);
-
-        assertThat(result).contains("minju, java");
+  public static final int LENGTH_OF_CARS = 3;
+
+  @Test
+  void 우승자를_가리는_기능() {
+    Judgement judgement = new Judgement();
+    List<Car> carLineUp = new ArrayList<>();
+    List<String> cars = Arrays.asList("pobi", "minju", "java");
+    List<Integer> progress = Arrays.asList(1, 2, 1);
+    List<String> winners;
+    String result;
+
+    for (int i = 0; i < LENGTH_OF_CARS; i++) {
+      carLineUp.add(new Car(cars.get(i), progress.get(i)));
     }
 
-    @Override
-    public void runMain() {
-        Application.main(new String[]{});
+    winners = judgement.judgeWinner(carLineUp);
+    result = String.join(", ", winners);
+    assertThat(result).contains("minju");
```

Comment:
여기서 결과값을 테스트하기 위해 `String.join` 을 사용해주신 것 같아요. 테스트 메서드에는 winners 에 정확하게 같은 순서로 값이 포함되어 있는지 확인해주는 이런 메서드도 있답니다! 유용한 메서드이니 알아두시면 좋을 것 같아요 😄

```suggestion
    result = String.join(", ", winners);
    assertThat(winners).containsExactly("minju");
```

Reply:
이 코드를 처음 작성할때가 테스트 코드를 작성하는 것이 처음이어서, 테스트 메서드를 다양하게 활용하지 못하고 전체적으로 contains()를 많이 사용했습니다.

이번 리뷰에서 다른 테스트 메서드들에 대해서 더 찾아볼 수 있었고 이에 내용을 수정했습니다. :)